python/metatensor-torch/tests/serialization.py

```diff
@@ -27,18 +28,40 @@ def check_tensor(tensor):
 
 
 def test_load():
-    loaded = metatensor.torch.load(
-        os.path.join(
-            os.path.dirname(__file__),
-            "..",
-            "..",
-            "..",
-            "metatensor",
-            "tests",
-            "data.npz",
-        ),
+    path = os.path.join(
+        os.path.dirname(__file__),
+        "..",
+        "..",
+        "..",
+        "metatensor",
+        "tests",
+        "data.npz",
+    )
```

Comment:
We could think using pathlib at some point.

Reply:
Not sure what this would give us? `/` operator overloading mainly, or is there something else?

I'm not sure I love`/` operator overloading  to be honest =)

Reply:
We could avoid the stacking of  ".." and write it like

```python
    path = Path(__file__).parents[3] / "metatensor" / "tests" / "data.npz"
```

I also don't know if I am a big fan of overloading though.